samples/java-shopping-cart-quickstart/.bundleignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
What is it that reads a `.bundleignore`-file, I never saw that before?

Reply:
It's the script that zips the quickstart downloads:

https://github.com/lightbend/akkaserverless-java-sdk/blob/425fff891357c60f75c2c1ddb8ec4e6ab40ddf7a/docs/bin/bundle.sh#L53

It will fail anyway if the file doesn't exist (as a reminder if adding a new quickstart).

We could also use the gitignore (would need to use all gitignores up to the git root), but this allows other files to be excluded as well.

Reply:
Ah, good to know :) I had just copied it over from the other quickstart and didn't yet test the zip download.

Reply:
Added a line to `docs/Makefile` to include bundling this quickstart as well.